### ISSUE TITLE
Advanced Search feedback by community champions

### ISSUE THREAD

**Issue by prakhyatox** [FAIRsharing/fairsharing.github.io#2269]
Orginial ticket: https://github.com/FAIRsharing/fairsharing.github.io/issues/2139

Below comments are from the community champions https://zulip.fairsharing.org/#narrow/stream/20-Community-Champions/topic/Advanced.20Search/near/54197

- [x] Add gitbook link in the popup about Advanced Search: https://fairsharing.gitbook.io/fairsharing/how-to/advanced-search

- [x] Currently, when the download button is clicked in after getting results in advancedSearch, a excel/csv file is generated with 3 fields. As, per the suggestion/feedback from one of the champions more fields can be added in the excel file. As per Lindsey Anderson: ```FAIRsharing DOI, homepage link, and any publication marked (toggled) for citation of the record would be nice to have in a download file```
- [x] Please could we have the initial version of an empty advanced search include the first (empty) group so that the user doesn't always need to click on Add Group to get started? Then it would look like this upon initial loading of an empty query builder
![image](https://github.com/FAIRsharing/fairsharing.github.io/assets/109139583/469e845f-009c-4490-b64a-2a2ebacab2bd)

- [x] Keeping  textbox to add search term
- [x] a FAIRsharing logo and introductory text at the top, as a header, on the query builder page - I realise this is already in the works but wanted to point out the feedback. Additionally, if that same introductory text could be added as a collapsible topic on the advanced search results page, just below the dark blue banner, that would be awesome.---```Didn't quite understood the comment but the heading and link to gitbook documentation is added in the header```
- [x] Many smaller multi-choice pulldowns (e.g. registry and record type rules) present their pulldowns as soon as you click on the white field box, while others (e.g. subjects) require that you start typing before any suggestions appear. They suggest that we either a) somehow indicate that you must start typing in order to start receiving suggestions, or b) provide all suggestions in a pulldown as soon as you click on the white box. The latter one we realise might be a bit difficult if there are many terms, e.g. subjects domains and taxonomies if it puts a large query burden / takes a long time for the query to run? --- ```Added text in tooltips of Autocomplete components```

**Comment by allysonlister** [comment]
> * a FAIRsharing logo and introductory text at the top, as a header, on the query builder page - I realise this is already in the works but wanted to point out the feedback. Additionally, if that same introductory text could be added as a collapsible topic on the advanced search results page, just below the dark blue banner, that would be awesome.---`Didn't quite understood the comment but the heading and link to gitbook documentation is added in the header`

In addition to what you've already done, just adding a FAIRsharing logo, centrally justified, above the title text in the advanced search popup please. Then, in an appropriate place such as next to the "download results" button, add the same sentence you added to the popup: "Find out more about our Advanced Search in our [gitbook documentation](https://fairsharing.gitbook.io/fairsharing/how-to/advanced-search)" e.g. with banner text or just regular paragraph text.

All other updates are great and I'll approve the PR once this is done. thanks!
